grafana/scylla-dash-per-server.master.json

```diff
@@ -675,7 +675,7 @@
                         },
                         "yaxes": [
                             {
-                                "format": "short",
+                                "format": "wps",
```

Comment:
scylla_storage_proxy_coordinator_foreground_writes presents current count of such writes, we don't take a time derivative of it, so "wps" (w per second) is not appropriate.